paladin将k8s configmap 生成的 "..data" 软链接文件夹识别成了配置文件

部署在k8s里的kratos项目，将配置文件替换为configmap时无法启动
报错如下：
```
panic: read configs/..data: is a directory
goroutine 1 [running]:
main.main()
	/go-mod/src/xxxxxx/cmd/main.go:19 +0x4e3
```
看了下这个文件夹是configmap生成的软链接，请问如何解决？

--- a/pkg/conf/paladin/file.go
+++ b/pkg/conf/paladin/file.go
@@ -167,7 +167,7 @@ func loadValues(base string) (map[string]*Value, error) {
 			return nil, fmt.Errorf("paladin: read dir %s error: %s", base, err)
 		}
 		for _, file := range files {
-			if !file.IsDir() {
+			if !file.IsDir() && (file.Mode()&os.ModeSymlink) != os.ModeSymlink {
 				paths = append(paths, path.Join(base, file.Name()))
 			}
 		}

